keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt

```diff
@@ -174,7 +174,7 @@ open class SqlResourceRepository(
         .from(EVENT)
         .where(EVENT.SCOPE.eq(Scope.APPLICATION.name))
         .and(EVENT.UID.eq(application))
-        .and(EVENT.TIMESTAMP.lessOrEqual(LocalDateTime.ofInstant(after, ZoneOffset.UTC)))
+        .and(EVENT.TIMESTAMP.greaterOrEqual(LocalDateTime.ofInstant(after, ZoneOffset.UTC)))
```

Comment:
This was another bug: the query was backwards! 😱

Reply:
hmm... didn't we send here the `until` time? how come it used to work?

Reply:
Yes, I just renamed `until` to `after` for clarity. But it turns out that the query never worked (there wasn’t a test for it, I should probably add one).

Reply:
ha I just saw that the only place we use this function is in `ActuationPauser:addApplicationActuationEvents` and you changed the logic there, so I guess it used to work with that query? :)

Reply:
No... that's what I meant, I think we just got lucky that there weren't many apps paused, or no one was paying attention to those events being present in the event history. 🤷‍♂️